gtfs-validator/main/src/test/java/org/mobilitydata/gtfsvalidator/validator/TransfersAreUniqueValidatorTest.java

```diff
@@ -0,0 +1,120 @@
+                                              .setCsvRowNumber(2)
+                                              .build();
+  private final GtfsTransfer transfer_2_duplicate = new GtfsTransfer.Builder()
+                                                        .setFromStopId("stop_3")
+                                                        .setToStopId("stop_7")
+                                                        .setCsvRowNumber(4)
+                                                        .build();
+  private final GtfsTransfer transfer_3 = new GtfsTransfer.Builder()
+                                              .setFromStopId("stop_5")
+                                              .setToStopId("stop_9")
+                                              .setCsvRowNumber(3)
+                                              .build();
+
+  @Test
+  public void allDifferentTransfersShouldNotGenerateNotice() {
+    final NoticeContainer noticeContainer = new NoticeContainer();
+    List<GtfsTransfer> transfers = ImmutableList.of(transfer_1, transfer_2, transfer_3);
+    validator.transferTable = GtfsTransferTableContainer.forEntities(transfers, noticeContainer);
+    validator.validate(noticeContainer);
+    assertThat(noticeContainer.getNotices()).isEmpty();
+  }
+
+  @Test
+  public void onlyDuplicateTransfersShouldGenerateNotice() {
+    final NoticeContainer noticeContainer = new NoticeContainer();
+    List<GtfsTransfer> transfers = ImmutableList.of(transfer_1, transfer_1_duplicate);
+    validator.transferTable = GtfsTransferTableContainer.forEntities(transfers, noticeContainer);
+    validator.validate(noticeContainer);
+    assertThat(noticeContainer.getNotices())
+        .containsExactly(new TransfersAreUniqueNotice(
+            /* fromStopId= */ "stop_1",
+            /* toStopId= */ "stop_2",
+            /* duplicateCsvRowNumber= */ 10,
+            /* originalCsvRowNumber= */ 1));
+  }
+
+  @Test
+  public void someDuplicateTransfersShouldGenerateNotice() {
+    final NoticeContainer noticeContainer = new NoticeContainer();
+    List<GtfsTransfer> transfers = ImmutableList.of(
+        transfer_1, transfer_2, transfer_3, transfer_1_duplicate, transfer_2_duplicate);
+    validator.transferTable = GtfsTransferTableContainer.forEntities(transfers, noticeContainer);
+    validator.validate(noticeContainer);
+    assertThat(noticeContainer.getNotices())
+        .containsExactly(new TransfersAreUniqueNotice(
+                             /* fromStopId= */ "stop_1",
+                             /* toStopId= */ "stop_2",
+                             /* duplicateCsvRowNumber= */ 10,
+                             /* originalCsvRowNumber= */ 1),
+            new TransfersAreUniqueNotice(
+                /* fromStopId= */ "stop_3",
+                /* fromStopId= */ "stop_7",
+                /* duplicateCsvRowNumber= */ 4,
+                /* originalCsvRowNumber= */ 2));
+  }
+
+  @Test
```

Comment:
One other thing you could test for is that reverse transfers don't trigger this notice. E.g. `stop_1 to stop_2` and `stop_2 to stop_1` shouldn't cause a notice. 

This would help prevent someone from breaking this behaviour with a future change to the implementation.

Reply:
done!